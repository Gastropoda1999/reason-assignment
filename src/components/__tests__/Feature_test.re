open Jest;
open Expect;

let _ =

describe("Feature", () => {
  test("renders checkmark emoji", () => {
    let tree = <Feature emoji=Feature.Checkmark text="A feature" />
    |> ReactShallowRenderer.renderWithRenderer;

  expect(tree) |> toMatchSnapshot
});

  test("renders soon emoji", () => {
    let tree = <Feature emoji=Feature.Soon text="A feature" />
      |> ReactShallowRenderer.renderWithRenderer;

    expect(tree) |> toMatchSnapshot;
  });
});
