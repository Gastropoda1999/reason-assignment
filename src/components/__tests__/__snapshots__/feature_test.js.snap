// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Feature renders checkmark emoji 1`] = `
<div
  className="css-pat51 css-vlxvfg"
>
  <Checkmark
    reasonProps={
      Array [
        Array [
          "Checkmark",
          [Function],
          Array [
            0,
          ],
          [Function],
          [Function],
          [Function],
          [Function],
          [Function],
          [Function],
          [Function],
          [Function],
          0,
          [Function],
          0,
        ],
      ]
    }
  />
  <h3>
    A feature
  </h3>
</div>
`;

exports[`Feature renders soon emoji 1`] = `
<div
  className="css-pat51 css-vlxvfg"
>
  <Soon
    reasonProps={
      Array [
        Array [
          "Soon",
          [Function],
          Array [
            0,
          ],
          [Function],
          [Function],
          [Function],
          [Function],
          [Function],
          [Function],
          [Function],
          [Function],
          0,
          [Function],
          0,
        ],
      ]
    }
  />
  <h3>
    A feature
  </h3>
</div>
`;
